packages/lodestar/src/sync/regular/naive/naive.ts

```diff
@@ -52,9 +57,17 @@ export class NaiveRegularSync implements IRegularSync {
     this.logger.info("Started regular syncing");
     this.chain.on("processedBlock", this.onProcessedBlock);
     this.currentTarget = this.chain.forkChoice.headBlockSlot();
+    const state = await this.chain.getHeadState();
+    this.slotAtStart = getCurrentSlot(this.config, state.genesisTime);
```

Comment:
Does this work if there are a lot of skip slots?

Why not start gossip as soon as regular sync starts?

Reply:
Gossip is started as soon as regular sync starts, this is all about listen for its event.
I'll change to listen in start bc I removed `onNewSlot` already, this would make it simpler